packages/client/src/webAPI/webAudioAPI.ts

```diff
@@ -0,0 +1,48 @@
+export default class AudioAPI {
+  private static _instance: AudioAPI
+  private _audioContext!: AudioContext
+  private _audioRef!: HTMLMediaElement
+  private _source!: MediaElementAudioSourceNode | null
+  private _gainNode!: GainNode | null
```

Comment:
Можно сразу инициализировать
```suggestion
  private _source!: MediaElementAudioSourceNode | null = null
  private _gainNode!: GainNode | null = null
```